mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestDerivates.java

```diff
@@ -394,29 +457,60 @@ boolean isDisplayEnabled() {
             return displayEnabled;
         }
 
-        @FormDataParam("display-enabled")
         @JsonProperty("display-enabled")
         public void setDisplayEnabled(boolean displayEnabled) {
             this.displayEnabled = displayEnabled;
         }
 
+        @FormParam("display-enabled")
+        @DefaultValue("true")
```

Comment:
the corresponding attribute in derivate xml is called "display" only - why is another name used here?

Reply:
it's derived from the method name "setDisplayEnabled", but I can rename it to display